### DISCUSSION TITLE
Non-hierarchical DDM for group data to get independant parameter estimates?

### DISCUSSION THREAD

**Discussion by naziajassim** [lnccbrown/HSSM#493] [Q&A] [answered]
Hi there, I was wondering if there's a simple way of setting up a non-hierarchical ddm after reading in my group data?  I have a heterogenous group of participants with different scores on a psychiatric questionnaire, so I'd like each participant's parameter estimates to be independant of the group. 

In order words, is there a simple trick to ensure non-hierarchical/within-subject ddm's are computed separately for each participant? Or would I have to read in each participant's data separately and run separate within-subject ddm's for each? 

Thanks a ton and looking forward to all the upcoming HSSM updates!

**Comment by frankmj** [comment, marked as answer]
You can certainly model each participant individually but you will not
benefit from hierarchical inference. It is still ok to fit hierarchical
models and then correlate posterior parameters from each individual with
the questionnaires (see here
<https://www.sciencedirect.com/science/article/pii/S0022249616300025>).
Even better you can use a between subjects regression to directly estimate
within the model how the scores affect a particular model parameter (but
for this approach it would be helpful to have a specific hypothesis about
which score relates to which parameter).

That said, if you still want to fit each participant individually, you can
simply create different data frames for each participant, define the model
applied to that dataframe, and then sample each of them.
